lib/backend_common/backend_common/auth.py

```diff
@@ -186,9 +186,9 @@ def __init__(self, claims, authenticated_email=None, permissions=[], token_data=
     def get_id(self):
         parts = ['token', self.claims['typ']]
         if 'jti' in self.claims:
-            parts.append('id={}'.format(self.claims['jti']))
+            parts.append(f"id={self.claims['jti']}")
```

Comment:
@La0 @garbas what is your opinion here? Should we 1) change the string enclosing quotes to " so that we can keep ' for the dict; 2) keep the string enclosing to ' and change the dict to "; 3) don't use f-strings when we have a dict?

Reply:
I vote 2